### PR TITLE
Adding atomneb

### PR DESCRIPTION
**Pull request by danehkar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by danehkar** [comment]
@conda-forge/staged-recipes
Ready to review!

**Comment by danehkar** [comment]
@conda-forge-linter ready for review

File: recipes/atomneb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - pip
+    - python >=3.8
+    - astropy
+    - numpy
+    - pandas
+  run:
+    - astropy
+    - numpy
+    - pandas
+    - python
```
Comment (chrisburr) [inline recipes/atomneb/meta.yaml:28]: This needs the same version constraint as in host

Comment (danehkar) [reply recipes/atomneb/meta.yaml]: @chrisburr I added the versions in both 'host' and 'run'.

File: recipes/atomneb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - astropy >=4.0
+    - numpy >=1.17
+    - pandas >=1.1
```
Comment (scopatz) [inline recipes/atomneb/meta.yaml:23]: You probably just need python and pip in host

Comment (danehkar) [reply recipes/atomneb/meta.yaml:23]: @scopatz I got the building errors in **linux_64**, **osx_64**, and **win_64** after removing them from 'host':

```
2020-10-05T20:48:29.8077779Z     ModuleNotFoundError: No module named 'pandas'
2020-10-05T20:48:29.8418496Z ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
.
.
.
2020-10-05T20:48:39.6337782Z ##[error]Bash exited with code '1'.
2020-10-05T20:48:39.6356149Z ##[section]Finishing: Run docker build
```

**Comment by danehkar** [comment]
@scopatz @chrisburr Please recheck the meta.yaml. I think that there are no issues now. Thanks!

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**